sdk/core/azure-core/azure/core/polling/_poller.py

```diff
@@ -102,7 +105,7 @@ def resource(self):
         return self._deserialization_callback(self._initial_response)
 
 
-class LROPoller(object):
+class LROPoller(Generic[HTTPRequestType, HTTPResponseType]):
```

Comment:
That's not correct, I know I said on issue #10605 that it should "look like this", but I was refering to "should use Generic", I didn't mean precise copy/pasting.
Please take a step back, zoom-out and understand what the poller is doing, and what should be written here